OsmSharp.IO.API/IClientsFactory.cs

```diff
@@ -18,15 +18,6 @@ public interface IClientsFactory
         /// <param name="password"></param>
         /// <returns></returns>
         IAuthClient CreateBasicAuthClient(string username, string password);
```

Comment:
I'm sorry I let this sit so long. The ticket also mentions that basic-auth should also be removed. Is there a reason do leave it? Want to do that at another time or in a separate PR?

Reply:
Oh yeah, missed that... Feel free to remove that.